src/components/ManagementDataSection/ManagementDataSection.module.css

```diff
@@ -1,18 +1,3 @@
-@import '../../common/vars.css';
-
-.export-button {
-  background-color: var(--color-dark-blue-muted);
-  color: var(--text-color-white);
-}
-
-.dropdown-button button {
-  color: var(--text-color-white);
-}
-
-.filter-button {
-  color: var(--text-color-white);
-}
-
```

Comment:
Why have all of these been removed?

Reply:
^ odd we didn't touch those :0 maybe during a merge?